app/experimenter/normandy/tasks.py

```diff
@@ -108,20 +110,43 @@ def update_launched_experiments():
     metrics.incr("update_launched_experiments.completed")
 
 
+def is_qaOnly(recipe_data):
+    if "filter_object" in recipe_data:
+        filter_objects = {f["type"]: f for f in recipe_data["filter_object"]}
```

Comment:
You know if you take `: f` out of this, it'll be a set comprehension instead of a dict comprehension and do the same thing as what you're doing here, checking membership of the key.  Not a requirement to change, just a tip ;)